### ISSUE TITLE
Organize list by rate of return

### ISSUE THREAD

**Issue by davecraige** [ong/awesome-decentralized-finance#25]
Love this list.  Would love to begin organizing the list by the rate of return that some of these DeFi projects provide.  I think the public will want to know which DeFi projects are the most profitable.  

There could also be a section on risk factors as well.

**Comment by atnickallen** [comment]
Agreed. But this is a very dynamic result that should be structured more like a price feed than a static list.
